project/AddMetaInfLicenseFiles.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+import org.mdedetrich.apache.sonatype.ApacheSonatypePlugin
+import org.mdedetrich.apache.sonatype.ApacheSonatypePlugin.autoImport.*
+import sbt.*
```

Comment:
Can you use `._` here instead of `.*`. Since this is an SBT plugin its using Scala 2.12 and Intellij incorrectly is not picking up the correct Scala dialect for sbt plugins.

Reply:
fixed that now - IntelliJ does this to files when you copy them around